src/db/z_db.erl

```diff
@@ -213,9 +221,11 @@ with_connection(F, Context) ->
 	end.
 
 
+-spec assoc_row(string(), #context{}) -> list(tuple()).
 assoc_row(Sql, Context) ->
     assoc_row(Sql, [], Context).
 
+-spec assoc_row(string(), list(tuple()), #context{}) -> list(tuple()).
```

Comment:
The spec in line 228 uses a record, please define a type for the record and use that instead.